packages/frontend/src/features/sqlRunner/components/BarChartAggregationConfig.tsx

```diff
@@ -58,18 +75,35 @@ export const BarChartAggregationConfig: FC<Props> = ({
     const aggregationOptionsWithNone = options ?? [];
 
     return (
-        <SegmentedControl
+        <Select
+            radius="md"
             data={aggregationOptionsWithNone.map((option) => ({
                 value: option,
-                label: (
-                    <Group noWrap spacing={0}>
-                        <AggregationIcon aggregation={option} />
-                        <Text>{capitalize(option)}</Text>
-                    </Group>
-                ),
+                label: capitalize(option),
             }))}
+            itemComponent={AggregationItem}
+            icon={aggregation && <AggregationIcon aggregation={aggregation} />}
             value={aggregation ?? aggregationOptionsWithNone?.[0]}
             onChange={(value) => value && onChangeAggregation(value)}
+            styles={(theme) => ({
+                input: {
+                    width: '110px',
```

Comment:
can't use any of mantine's size keywords - this will do

Reply:
As soon as you do something a tiny bit custom...